### PR TITLE
Add details on how to remove Redash

### LINKED ISSUE #27: Add documentation about fixing docker-compose after removing redash
After removing redash from a server any use of docker-compose yields the following error:

```
ERROR: .FileNotFoundError: [Errno 2] No such file or directory: '/opt/redash/docker-compose.yml'
```

This effectively makes docker-compose unusable for any other purpose. The cause is these two lines in the user's  `.profile` file which are added by the setup script.

```
export COMPOSE_PROJECT_NAME=redash
export COMPOSE_FILE=/opt/redash/docker-compose.yml
```

It would be useful if this were documented in the README.

### PR DESCRIPTION
**Pull request by johnbullnz**
Fixes #27

### CODE REVIEW

**Review by susodapop — APPROVED** [review]
Thanks for this!

File: README.md [hunk 1]
```diff
@@ -21,3 +21,16 @@ See [Upgrade Guide](https://redash.io/help/open-source/admin-guide/how-to-upgrad
 ### How do I use `setup.sh` on a different operating system?
 
 You will need to update the `install_docker` function and maybe other functions as well.
+
+### How do I remove Redash if I not longer need it?
```
Comment (peter279k) [inline README.md:25]: Fix typo: `I no longer need it`.

**Comment by snickerjp** [comment]
I think it is a good suggestion.

File: README.md (hunk 1 above)
Comment (snickerjp) [reply README.md:25]: fixed: #57

**Pull request merged**